packages/@lwc/integration-karma/test/synthetic-shadow/scoped-id/multiple-idrefs.spec.js

```diff
@@ -18,8 +18,8 @@ it('should handle multiple idrefs when set dynamically', () => {
         expect(hokkaido.id).toMatch(/^hokkaido-\w+/);
     }
 
-    expect(input.ariaLabelledBy).toContain(aomori.id);
-    expect(input.ariaLabelledBy).toContain(hokkaido.id);
+    expect(input.getAttribute('aria-labelledby')).toContain(aomori.id);
+    expect(input.getAttribute('aria-labelledby')).toContain(hokkaido.id);
```

Comment:
This is exactly the observable change that we would introduce, _when_ the flag is set to true. (I.e. Arbitrary `Element`s would no longer have non-standard `aria*` property reflection.)